crates/router/src/connector/utils.rs

```diff
@@ -352,6 +370,102 @@ pub fn get_header_key_value<'a>(
         ))?
 }
 
+pub fn to_boolean(string: String) -> bool {
```

Comment:
what is the requirement for this? I see this function is not being used anywhere, can be removed

Reply:
it is being used in nuvei.rs::695